src/mergeLinkedList/mergeLinkedListTest.java

```diff
@@ -0,0 +1,50 @@
+package mergeLinkedList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class mergeLinkedListTest {
+
+    @org.junit.jupiter.api.Test
+    void mergeLinkedList() {
```

Comment:
You test names need to be more informative about what the test is actually testing.  I know I've told you this before, please start implementing!  :)